Dockerfile

```diff
@@ -1,18 +1,31 @@
 FROM rust:1-alpine3.20 AS builder
 ENV RUSTFLAGS="-C target-feature=-crt-static -C target-cpu=native"
 RUN apk add --no-cache musl-dev
+
 WORKDIR /pumpkin
 COPY . /pumpkin
+
+# build release
 RUN --mount=type=cache,sharing=private,target=/pumpkin/target \
     --mount=type=cache,target=/usr/local/cargo/git/db \
     --mount=type=cache,target=/usr/local/cargo/registry/ \
     cargo build --release && cp target/release/pumpkin ./pumpkin.release
+
+# strip debug symbols from binary
 RUN strip pumpkin.release
 
 FROM alpine:3.20
-WORKDIR /pumpkin
+
+# Identifying information for registries like ghcr.io
+LABEL org.opencontainers.image.source=https://github.com/Snowiiii/Pumpkin
+
 RUN apk add --no-cache libgcc
+
 COPY --from=builder /pumpkin/pumpkin.release /pumpkin/pumpkin
+
+# set workdir to /config 
+WORKDIR /config
```

Comment:
I do not like this comment. The code is self explanatory.

Reply:
Sure I can agree on that one.

To be honest this `/config`, is still up for debate as it doesn't quite fit the logic for future dynamic content (world, logs, datapacks, etc...).

I'm open for discussion to get a more appropriate name (ex. `/app` or `/server`).

Reply:
I'm not too familiar with docker, but I think if I've understood the issue correctly is that for some reason when launching docker, it doesn't automatically generate the config files like it does when you run the binary normally? I think `/config` sounds appropriate due to the files being configuration files

Reply:
> I'm not too familiar with docker, but I think if I've understood the issue correctly is that for some reason when launching docker, it doesn't automatically generate the config files like it does when you run the binary normally? I think `/config` sounds appropriate due to the files being configuration files

I'd like to add those are the files the server creates implicitly, these two files are made in the root. In the future the world folder, datapack folders, logs folders, ... will live next to those two files. Making the `/config` folder a less ideal option IMO. As such I'm trying to thing a bit further into the future and generalize the root folder name.

Reply:
Comment should be resolved, IMO. Open for further discussion.